Automation rules: add delete version action (#7644)
readthedocs/readthedocs.org: docs/automation-rules.rst

   The stable version is also automatically updated at the same time.
   See more in :doc:`versions`.

.. note::

>    Default versions aren't deleted even if they match a rule.

Reviewer: ```suggestion
   Default version isn't deleted even if it matches a rule.
```

We have only one default version, right?

   You can use the ``Set version as default`` action to change the default version

>    before deleting the current one.

Reviewer: I'd move this note above the previous one to keep it close to "Delete version". Even, if possible, I'd put it inside the bullet.

Otherwise, it's not easy to realize that this note is strictly related to "Delete version" action.

- Version type: ``Branch``
- Action: ``Activate version``

> Automatically delete an active version of a deleted branch

Reviewer: ```suggestion
Delete an active version when a branch is deleted
```

There is no need to mention "Automatically". Also, I re-wrote the sentence. I think it's clearer.
